.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,159 @@
+      - [@azure/msal-browser](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-browser/FAQ.md)
+      - [@azure/msal-node](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-node/FAQ.md)
+      - [@azure/msal-angular](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-angular/FAQ.md)
+      - [@azure/msal-react](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-react/FAQ.md)
+      - [@azure/msal or msal](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-core/docs/FAQ.md)
+      
+      If you have questions about timelines, please see our [roadmap](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/roadmap.md). Issues pertaining to timelines may be closed without comment.
+
+- type: markdown
+  attributes:
+    value: |
+      ## Issue Details
+    
+- type: dropdown
+  attributes:
+    label: Core Library
+    options: 
+      - "@azure/msal or msal"
+      - "@azure/msal-browser"
+      - "@azure/msal-node"
+  validations:
+    required: true
+    
+- type: input
+  attributes:
+    label: Core Library Version
+    description: "Please enter the version you are using."
+    placeholder: "2.13.0"
+  validations:
+    required: true
+    
+- type: dropdown
+  attributes:
+    label: Wrapper Library
+    options:
+      - "@azure/msal-angular"
+      - "@azure/msal-react"
+    
+- type: input
+  attributes:
+    label: Wrapper Library Version
+    description: "Please enter the version you are using."
+    placeholder: "2.0.0-beta.0"
+    
+- type: textarea
+  attributes:
+    label: Description
+    description: "Please briefly describe your issue."
+  validations: 
+    required: true
+    
+- type: textarea
+  attributes:
+    label: Error Message
+    description: "Please provide any error message you are receiving and a stack trace."
+    
+- type: textarea
+  attributes:
+    label: MSAL Configuration
+    description: "Please provide your MSAL configuration."
+    render: javascript
+    placeholder: |
+      {
+        auth: {
+          clientId: "clientId",
+          authority: "https://login.microsoftonline.com/common/"
+        },
+        cache: {
+          cacheLocation: "sessionStorage"
+          storeAuthStateInCookie: false
+        }
+      }
+  validations:
+    required: true
+    
+- type: textarea
+  attributes:
+    label: Relevant Code Snippets
+    description: "Please provide relevant code snippets that can be used to reproduce this issue."
+    render: javascript
+  validations:
+    required: true
+    
+- type: textarea
+  attributes:
+    label: Reproduction Steps
+    description: "Please provide steps to reproduce or a link to a sample which demonstrates this behavior."
+    placeholder: |
+      1.
+      2.
+      3.
+  validations:
+    required: true
```

Comment:
I think we should users to indicate if they have tried reproducing this issues with the latest version of their library.

Reply:
What if we update the prompt for the version field to say "Please indicate the latest version this can be reproduced in" and add a feature to the bot incrementally that auto comments on issues with a link to npm/changelog if they indicate they are using an older version? I'm not sure I see a whole lot of value in a simple checkbox tbh